lib/makeSchema.js

```diff
@@ -36,10 +36,10 @@ function containBlocks(opts) {
          * Wraps the given nodes in a default block
          * @param {List<Nodes>} value.toWrap
          */
-        normalize(transform, node, value) {
-            return value.toWrap.reduce((tr, n) => {
-                return tr.wrapBlockByKey(n.key, opts.typeDefault);
-            }, transform);
+        normalize(change, node, value) {
+            return value.toWrap.reduce((c, n) => {
```

Comment:
Hey @thomaswhyyou reduce loops are bad in the new slate since change is mutable now, this might be why one of your tests is failing?

Reply:
Hey @YurkaninRyan, thanks for taking a look! :) I refactored to prefer forEach instead of reduce but still not passing due to the same error :\

For some reason, assertDescendant(key) inside wrapBlockByKey() is throwing, and can't figure out why..